packages/mapviewer/src/modules/i18n/locales/de.json

```diff
@@ -513,12 +513,16 @@
     "print_scale": "Massstab",
     "print_underway_message": "Wir bitten Sie um Geduld. Ein mehrseitiger Druck wird erstellt. Dies nimmt einige Zeit in Anspruch.",
     "problem_announcement": "Problem melden",
+    "profile_could_not_generate": "Fehler: Das Profil konnte nicht erstellt werden",
     "profile_distance": "Luftlinie",
+    "profile_download_csv": "Daten als CSV-Datei abrufen",
     "profile_elevation_difference": "Höhendifferenz Start-Ende",
     "profile_elevation_down": "Abstieg",
     "profile_elevation_up": "Aufstieg",
     "profile_hike_time": "Marschzeit",
+    "profile_invert": "Gegenrichtung",
     "profile_no_data": "kein Datum",
+    "profile_out_of_bounds": "Objekte sind außerhalb der Grenzen, keine Profildaten konnten wiederhergestellt werden",
```

Comment:
In Switzerland, we don't use the ß 😀 (We don't even know the rules!). So you can easily put "ss" here.